### PR TITLE
az-*: add Spanish translation

### PR DESCRIPTION
**Pull request by crixodia**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/az-sshkey.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# az sshkey
+
+> Administra claves públicas SSH.
```
Comment (navarroaxel) [inline pages.es/common/az-sshkey.md:3]: Based on https://learn.microsoft.com/es-es/cli/azure/sshkey?view=azure-cli-latest
```suggestion
> Administra claves públicas SSH con máquinas virtuales.
```
To update on the English page:
> Manage ssh public key with virtual machines.

File: pages.es/common/az-vm.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# az vm
+
+> Administra máquinas virtuales en Azure.
+> Parte de `az`, el cliente de línea de comandos para Microsoft Azure.
```
Comment (navarroaxel) [inline pages.es/common/az-vm.md:4]: This should be updated in the English page too.
```suggestion
> Parte de `azure-cli`.
```

File: pages.es/common/az-vm.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+> Más información: <https://learn.microsoft.com/cli/azure/vm>.
+
+- Lista los detalles de las máquinas virtuales disponibles:
+
+`az vm list`
+
+- Crea una máquina virtual `UbuntuServer 18.04 LTS` y genera claves ssh:
+
+`az vm create --resource-group {{grupo_de_recursos}} --name {{nombre}} --image {{Canonical:UbuntuServer:18.04-LTS:latest}} --admin-user {{usuario_azure}} --generate-ssh-keys`
```
Comment (navarroaxel) [inline pages.es/common/az-vm.md:13]: Based on the first of this list https://learn.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az-vm-create-examples we can update this example to be out of time. This should be updated on the English page too
```suggestion
`az vm create --resource-group {{grupo_de_recursos}} --name {{nombre}} --image {{UbuntuLTS}} --admin-user {{usuario_azure}} --generate-ssh-keys`
```

File: pages.es/common/az-vm.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+> Administra máquinas virtuales en Azure.
+> Parte de `az`, el cliente de línea de comandos para Microsoft Azure.
+> Más información: <https://learn.microsoft.com/cli/azure/vm>.
+
+- Lista los detalles de las máquinas virtuales disponibles:
+
+`az vm list`
+
+- Crea una máquina virtual `UbuntuServer 18.04 LTS` y genera claves ssh:
```
Comment (navarroaxel) [inline pages.es/common/az-vm.md:11]:
```suggestion
- Crea una máquina virtual usando la imagen por defecto de Ubuntu y genera claves ssh:
```
For the English page:
> Create a virtual machine using the default Ubuntu image and generate ssh keys:

**Review by navarroaxel — APPROVED** [review]

File: pages/common/az-vm.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
 # az vm
 
 > Manage virtual machines in Azure.
-> Part of `az`, the command-line client for Microsoft Azure.
+> Part of `aazure-cli`.
```
Comment (kbdharun) [inline pages/common/az-vm.md:4]:
```suggestion
> Part of `azure-cli`.
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**